velox/functions/prestosql/MapFromEntries.cpp

```diff
@@ -127,14 +128,31 @@ class MapFromEntriesFunction : public exec::VectorFunction {
       wrappedKeys = valueRowVector->childAt(0);
       wrappedValues = valueRowVector->childAt(1);
     } else {
-      wrappedKeys = decodedValueVector->wrap(
-          valueRowVector->childAt(0),
-          *inputValueVector,
-          inputValueVector->size());
-      wrappedValues = decodedValueVector->wrap(
-          valueRowVector->childAt(1),
-          *inputValueVector,
-          inputValueVector->size());
+      auto indices =
```

Comment:
nit: can you add a follow up issue and add a reference to it in a TODO here?

Reply:
I am not sure if we want to use decoded->wrap anyway since its not recommended to be used outside the decoded vector anyway. Not sure if we need any todo here